### DISCUSSION TITLE
Immich takeout?

### DISCUSSION THREAD

**Discussion by amgschnappi** [immich-app/immich#9354] [Q&A]
Similar to google takeout, does immich offer immich takeout?

**Comment by alextran1502** [comment]
Not at the moment, since you already have access to the original file at UPLOAD_LOCATION

**Comment by amgschnappi** [comment]
Yeah. I have access to a mess. Not a logical folder structure.

**Reply by mmomjian** [reply]
You should enable the storage template: https://immich.app/docs/administration/storage-template/
